src/elements/content-explorer/stories/tests/MetadataView-visual.stories.tsx

```diff
@@ -5,45 +5,61 @@ import ContentExplorer from '../../ContentExplorer';
 import { DEFAULT_HOSTNAME_API } from '../../../../constants';
 import { mockMetadata, mockSchema } from '../../../common/__mocks__/mockMetadata';
 
-const EID = '0';
-const templateName = 'templateName';
-const metadataSource = `enterprise_${EID}.${templateName}`;
-const metadataSourceFieldName = `metadata.${metadataSource}`;
+// The intent behind relying on mockMetadata is to allow a developer to paste in their own metadata template schema for use with live API calls.
+const { scope: templateScope, templateKey } = mockSchema;
```

Comment:
not sure I understand this comment. is it saying that the developer should paste their template schema in `common/__mocks__/mockMetadata`? or is it referring to the storybook controls?

Reply:
a developer as in one of the maintainers of the element. this means that as we add new features, and we want to test metadata view with live API calls, we can replace mockSchema with our real schema and not have to modify too much of the rest of the test, just comment out MSW.

this does not mean developer as in consumer of the element

Reply:
that flow sounds better to be done through the storybook controls and as a regular story rather than a visual regression test

from what i understand, this particular story uses mocks because the token doesn't support metadata api calls. also in general VRTs should use mocked data